src/languages/en-US/commands/notifications.json

```diff
@@ -0,0 +1,66 @@
+{
+	"youTubeSubscriptionDescription": "Manage the server's YouTube subscriptions.",
+	"youTubeSubscriptionExtended": {
+		"usages": [
+			"set upload/live channel Channel",
+			"set upload/live message Content",
+			"add/subscribe ChannelUrl",
+			"remove/unsubscribe ChannelUrl",
+			"clear",
+			"list",
+			""
+		],
+		"extendedHelp": "Manage the YouTube subscriptions for this server.",
+		"explainedUsage": [
+			[
+				"set upload/live channel Channel",
+				"The channel to send the upload or live notifications."
+			],
+			[
+				"set upload/live message Content",
+				"The message to send in the specified channel."
+			],
+			[
+				"add/subscribe ChannelUrl",
+				"Adds a new subscription."
+			],
+			[
+				"remove/unsubscribe ChannelUrl",
+				"Removes an existing subscription."
+			],
+			[
+				"clear",
+				"Clears all existing subscriptions."
+			],
+			[
+				"list",
+				"Lists all the subscriptions."
+			]
+		],
+		"examples": [
+			"set upload channel #notifications",
+			"set upload message {{channel.name}} has uploaded a new video `{{title}}` at {{time}}, go check it out! {{url}}",
```

Comment:
as discussed on discord, these `{{...}}` get parsed out so we need to use different indicators here.